torchtext/data/dataset.py

```diff
@@ -315,7 +313,7 @@ def stratify(examples, strata_field):
     # The field has to be hashable otherwise this doesn't work
     # There's two iterations over the whole dataset here, which can be
     # reduced to just one if a dedicated method for stratified splitting is used
-    unique_strata = set(getattr(example, strata_field) for example in examples)
```

Comment:
Is this relevant to the drop of python2 support?

Reply:
Yes, it's using Python 3 syntax that was not possible when also supporting Python 2.